src/api/api/BIOMD/BIOMD.js

```diff
@@ -416,3 +423,171 @@ module.exports.LIST_VENDORS = async function (req, dbClient) {
 
   return res;
 };
+
+module.exports.RECORDS = async function (req, dbClient) {
+
+  try {
+    //Confirm Packet Received
+    console.log(
+      (await TIMESTAMP()) +
+      `: RCU-BIOMD-I001 : RECORDS processing request packet ID: ${req.ID}`
+    );
+
+    //Copy Requester Information
+    var res = Object.assign({}, req);
+
+    //req object defintion
+    var find = Object.assign({}, req.Request.find);
```

Comment:
Could we change the schema for find? Remove the variable find and just have reportType as the top level variable?